<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>backup</class>
 <widget class="QWidget" name="backup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="res_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="origin_res">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bu_res">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="res_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>源路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="bu_label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>备份路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>备份功能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bu_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bu">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>进行备份</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>323</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="return_main">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="directory_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pwd_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>加密密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>res_button</zorder>
  <zorder>origin_res</zorder>
  <zorder>bu_res</zorder>
  <zorder>res_label</zorder>
  <zorder>bu_label</zorder>
  <zorder>title</zorder>
  <zorder>bu_button</zorder>
  <zorder>bu</zorder>
  <zorder>return_main</zorder>
  <zorder>directory_button</zorder>
  <zorder>pwd_button</zorder>
  <zorder>password_label</zorder>
  <zorder>password</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
